### PR TITLE
i18n(es): Updated `i18n.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `i18n.mdx`
* [`fdc18b5`](https://github.com/withastro/starlight/commit/fdc18b5476957f8017a0fa1489c6fed89d5a9480)
* [`af2e43c`](https://github.com/withastro/starlight/commit/af2e43c7325a8f7fa6c9f867a3ec864daae39e96)

### CODE REVIEW

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 1]
```diff
@@ -140,7 +140,7 @@ Si no hay una traducción disponible para un idioma, Starlight mostrará a los l
 
 ## Traduce la UI de Starlight
 
-Starlight proporciona de forma predeterminada traducciones para los textos utilizados en la UI al inglés, checo, francés, alemán, italiano, japonés, holandés, portugués y español. También damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+Starlight proporciona de forma predeterminada traducciones para los textos utilizados en la UI al inglés, francés, alemán, japonés, portugués, danés, español y turco. También damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
```
Comment (delucis) [inline docs/src/content/docs/es/guides/i18n.mdx:143]: Hmm, looks like a few languages went missing here, no?
```suggestion
Starlight proporciona de forma predeterminada traducciones para los textos utilizados en la UI al inglés, checo, francés, alemán, italiano, japonés, holandés, portugués, danés, español y turco. También damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
```

**Comment by Waxer59** [comment]
Done!, thanks for the review!

File: docs/src/content/docs/es/guides/i18n.mdx (hunk 1 above)
Comment (Waxer59) [reply docs/src/content/docs/es/guides/i18n.mdx]: I didn't realize when updating it, it seems that the pull was not done well in my srry pc. 😅😅

**Review by delucis — APPROVED** [review]
Thank you @Waxer59!

**Pull request merged**